src/win/win_path_manager.h

```diff
@@ -5,12 +5,16 @@
 #include "i_path_manager.h"
 
 class WinPathManager : public IPathManager {
+    private: 
+        const char *appdata = getenv("APPDATA");
+        std::string path = appdata;        
     public:
         WinPathManager(){}
         ~WinPathManager(){}
         void initExecutablePath(const std::string called_cmd);
 
         static std::string executablePath;
+        std::string configPath = path.append("\\QuickKey");

```

Comment:
Make this static to provide global access. My aim was to prevent creating an instance of the path manager, as to have easy access to all paths.